### PR TITLE
Added more DNS Resource Record Types.

### PR DESCRIPTION
**Pull request by maxtors**
Added some more DNS RRTYPE's. Some are obsolete and or experimental, but having detection for these could help. Saw a few "in the wild" that should be obsolete, but they are still in use.

### CODE REVIEW

File: src/app-layer-dns-common.c [hunk 1]
```diff
@@ -1016,51 +1016,174 @@ void DNSCreateTypeString(uint16_t type, char *str, size_t str_size)
         case DNS_RECORD_TYPE_NS:
             snprintf(str, str_size, "NS");
             break;
-        case DNS_RECORD_TYPE_AAAA:
```
Comment (inliniac) [inline src/app-layer-dns-common.c]: I think the most used types should be on top for efficiency.

Comment (maxtors) [inline src/app-layer-dns-common.c]: That sounds smart, will update and submit a new PR soon.

Comment (inliniac) [inline src/app-layer-dns-common.c]: On 28-04-16 14:52, Andreas Moe wrote:
> In src/app-layer-dns-common.c
> <https://github.com/inliniac/suricata/pull/2041#discussion_r61421248>:
> 
>> @@ -1016,51 +1016,174 @@ void DNSCreateTypeString(uint16_t type, char *str, size_t str_size)
>>          case DNS_RECORD_TYPE_NS:
>>              snprintf(str, str_size, "NS");
>>              break;
>> -        case DNS_RECORD_TYPE_AAAA:
> 
> That sounds smart, will update and submit a new PR soon.

Thanks!

**Comment by maxtors** [comment]
Opened new PR #2042 .

**Pull request closed without merging**